.gitignore

```diff
@@ -25,3 +25,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+#Ignor DS_Store
+**/.DS_Store
```

Comment:
It's good idea to setup [global git ignore](https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) for such files.